src/screens/Home.js

```diff
@@ -26,7 +25,72 @@ const Card = ({ title }) => {
     );
 };
 
+const modules = [
```

Comment:
Why are you adding modules in home screen ?
You need to handle this in splash screen now.